Cabal/src/Distribution/Simple/Program/Internal.hs

```diff
@@ -17,21 +17,28 @@ import Prelude ()
 
 -- | Extract the version number from the output of 'strip --version'.
 --
--- Invoking "strip --version" gives very inconsistent results. We ignore
--- everything in parentheses (see #2497), look for the first word that starts
--- with a number, and try parsing out the first two components of it. Non-GNU
--- 'strip' doesn't appear to have a version flag.
+-- Invoking "strip --version" gives very inconsistent results. We
+-- ignore everything in parentheses (see #2497), look for the first
+-- word that starts with a number, and try parsing out the first two
+-- components of it. Non-GNU, non-LLVM 'strip' doesn't appear to have
+-- a version flag.
 stripExtractVersion :: String -> String
 stripExtractVersion str =
   let numeric "" = False
       numeric (x : _) = isDigit x
 
+      closingParentheses =
+        [ ")"
+        , -- LLVM strip outputs "llvm-strip, compatible with GNU strip\nLLVM (http://llvm.org/):\n..."
+          "):"
```

Comment:
I am not sure, why are we checking for `):` too?

I mean, I cannot understand why this is necessary form the PR description.

Reply:
It is explained one line above. The logic omits `words` that start with `(` and end with `)`, but llvm-strip has a word that starts with `(` and ends with `):`